src/parser/parse_tree.h

```diff
@@ -1928,6 +1930,7 @@ struct pt_attr_def_info
 {
   PT_NODE *attr_name;		/* PT_NAME */
   PT_NODE *data_default;	/* PT_DATA_DEFAULT */
+  PT_NODE *on_update;
```

Comment:
on_update node could be replaced by DB_DEFAULT_EXPR_TYPE. It is its only used field. As consequnce, pt_data_on_update_info, PT_ON_UPDATE_INFO, PT_ON_UPDATE and all specific functions (init, apply, print) may be removed.

note on printing: pt_print_attr_def should also consider on_update while printing.